spec/transport_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'spec_helper'
+
+class DummyTransport < Transport
+  def initialize
+    @available = true
+  end
+end
+
+RSpec.describe DummyTransport do  
+
+  describe '#toggle_available!' do
+    context 'when toggle availability' do
+      it { expect(subject.toggle_available!).to be false }      
```

Comment:
Such case is better to test with `change` [matcher](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/change-matcher)

Reply:
rewrited test with change matcher